go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -84,8 +84,12 @@ func (agent *ActionAgent) Backup(ctx context.Context, concurrency int, logger lo
 	returnErr := mysqlctl.Backup(ctx, agent.Cnf, agent.MysqlDaemon, l, dir, name, concurrency, agent.hookExtraEnv())
 
 	if builtin != nil {
+
+		// Starting from here we won't be able to recover if we get stopped by a cancelled
+		// context. Thus we use the background context to get through to the finish.
+
```

Comment:
Can you create a separate `ctx := context.Background()` here and use it in both places.
Ideally, they say you have to use `context.TODO()` (because you're not supposed to create background contexts outside of main.go). But I'm not very hung up on that rule.

Reply:
Done. I squashed it with the previous commit.